### PR TITLE
alephone (update): Use latest version game data.

### PR DESCRIPTION
**Pull request by s1eve-mcdichae1**
(Unless using old-version game engine, then use old-version data too.)

### CODE REVIEW

**Comment by cmitu** [comment]
Looks ok, thank you.

File: scriptmodules/ports/alephone.sh [hunk 1]
```diff
@@ -54,18 +54,22 @@ function install_alephone() {
 }
 
 function game_data_alephone() {
-    local release_url="https://github.com/Aleph-One-Marathon/alephone/releases/download/release-20150620"
+    local version="20150620"
+    if compareVersions "$__os_debian_ver" ge 9 || [[ -n "$__os_ubuntu_ver" ]]; then
+        local version="20220115"
```
Comment (joolswills) [inline scriptmodules/ports/alephone.sh:59]: This doesn't need the additional "local".

Comment (s1eve-mcdichae1) [reply scriptmodules/ports/alephone.sh]: You're right, of course. Thanks.

**Comment by joolswills** [comment]
I squashed the commits. Thanks.

**Pull request merged**